org.gtk.Gtk3theme.Mojave-light.json

```diff
@@ -0,0 +1,43 @@
+{
+  "id": "org.gtk.Gtk3theme.Mojave-light",
+  "branch": "3.22",
+  "runtime": "org.freedesktop.Sdk",
+  "build-extension": true,
+  "sdk": "org.freedesktop.Sdk",
+  "runtime-version": "18.08",
```

Comment:
I'm pretty sure branch and runtime-version are not right.

Reply:
Could you remind me what they should be referring to? It’s been a while since I wrote this. Thanks!

Reply:
forget what I said about branch I was incorrect, but `runtime-version` should probably "20.08".

Reply:
Thanks